internal/store/static.go

```diff
@@ -114,11 +114,29 @@ func (store *StaticStore) GetPlugins(pluginFilter *model.PluginFilter) ([]*model
 }
 
 // getPlugins returns all plugins compatible with the given server version, sorted by name ascending.
-func (store *StaticStore) getPlugins(serverVersion string) ([]*model.Plugin, error) {
+func (store *StaticStore) getPlugins(serverVersion string, includeEnterprisePlugins bool) ([]*model.Plugin, error) {
 	var result []*model.Plugin
 	plugins := map[string]*model.Plugin{}
 
 	for _, storePlugin := range store.plugins {
+		if storePlugin.Enterprise && !includeEnterprisePlugins {
+			if serverVersion == "" {
```

Comment:
`serverVersion == ""` means no server version restriction, right? in which case shouldn't we include enterprise plugins? as we do for server versions < 5.25.0?

Reply:
I think this is the case where we exclude enterprise plugins if requested by a client that doesn't specify a server version (i.e. make the `EnterprisePlugins` flag "real")

Reply:
Ahh right, in this case we _should_ be honouring the flag.

Reply:
> I think this is the case where we exclude enterprise plugins if requested by a client that doesn't specify a server version (i.e. make the `EnterprisePlugins` flag "real")

Exactly